servers/src/main/java/tachyon/worker/block/BlockMasterSync.java

```diff
@@ -217,4 +218,33 @@ private void resetMasterClient() {
         new MasterClient(NetworkAddressUtils.getMasterAddress(mTachyonConf),
             mMasterClientExecutorService, mTachyonConf);
   }
+
+  /**
+   * Thread to remove block from master
+   */
+  private class BlockRemover implements Runnable {
+    private BlockDataManager mBlockDataManager;
+    private long mUserId;
+    private long mBlockId;
+
+    public BlockRemover( BlockDataManager mBlockDataManager, long userId, long blockId) {
+      this.mBlockDataManager = mBlockDataManager;
+      this.mUserId = userId;
+      this.mBlockId = blockId;
```

Comment:
We avoid using `this` in Tachyon, could you instead name the arguments like `blockDataManager`?